src/content-helper/excerpt-generator/components/excerpt-panel.tsx

```diff
@@ -0,0 +1,197 @@
+/**
+ * External dependencies
```

Comment:
```suggestion
 * WordPress dependencies
```

Conventionally, we've been using `WordPress dependencies` when imports are only coming from `@wordpress` packages. There are a couple of additional occurrences of this.